src/disclosures/js/views/metric-view.js

```diff
@@ -5,94 +5,113 @@ var getSchool = require( '../dispatchers/get-school-values' );
 var formatUSD = require( 'format-usd' );
 
 var metricView = {
+  metrics: {
+    debtBurden: {
+      school: NaN,
```

Comment:
I think it's best to assume there's no data available, which means no number (rather than showing an inaccurate default number).